src/app/zones/locales/en-us/index.yaml

```diff
@@ -205,9 +205,8 @@ zones:
         title: 'An error occurred while creating the Zone {zoneName}'
         description: 'You may retry this operation.'
   delete:
-    confirmModal:
+    confirm_modal:
+      proceed_button: 'Yes, delete'
       text1: 'Are you sure you want to delete the Zone {zoneName}?'
       text2: 'This action cannot be reversed.'
-      proceedText: 'Yes, delete'
       title: 'Delete Zone'
-      errorText: 'An unexpected error occurred'
```

Comment:
I feel like a lot more of this "delete modal" text could just go elsewhere (in common or component.delete-resource-modal)? `Delete {type}`, `Are you sure you want to delete the {type} {name}?` etc.

I would think we want this kinda stuff to be consistent between "entities"?